### ISSUE TITLE
Rename `fab` to `fabrikate`

### ISSUE THREAD

**Issue by evanlouie** [microsoft/fabrikate#243]
**As a:** Operator

**I want:** References to `fab` to be `fabrikate` instead

**So that:** the binary that gets automatically injected into my path via `go get -u -v github.com/microsoft/fabrikate` matches what is referenced in docs and in fabrikate release downloads.

**Describe the solution you'd like:** 

upon release of v1.0.0

- rename all instances of `fab` to `fabrikate` in docs
- change name of generated binary made for releases to `fabrikate` instead of `fab` 

**Describe alternatives you've considered:**

**Additional context:**

**Comment by evanlouie** [comment]
@timfpark thoughts?

**Comment by timfpark** [comment]
yes - let's standardize on `fabrikate` as the name of the tool everywhere.
